articles/ai-services/content-safety/quickstart-protected-material.md

```diff
@@ -30,7 +30,7 @@ The following section walks through a sample request with cURL. Paste the comman
 1. Replace `<your_subscription_key>` with one of the keys that come with your resource.
 1. Optionally, replace the `"text"` field in the body with your own text you'd like to analyze.
     > [!TIP]
-    > See [Input requirements](./overview.md#input-requirements) for maximum text length limitations. Protected material detection is meant to be run on LLM completions, not user prompts.
+    > See [Input requirements](./overview.md#input-requirements) for maximum text length limitations. Protected material detection is meant to be run on LLM completions, not user prompts. Protected material only work if length is > 110 characters, better than 200 characters; otherwise, the detection result will always be "False".
```

Comment:
let's keep the input requirements in the input requirements section, to avoid that information being spread out and hard to track